tensorflow/core/ops/string_ops.cc

```diff
@@ -247,6 +247,57 @@ shape: a length-2 vector of int64 representing the shape of the sparse
   of tokens in a single input entry.
 )doc");
 
+REGISTER_OP("StringUTF8Split")
```

Comment:
Nit: Can we call this `StringSplitUTF8` instead?

Reply:
Done.